uk.co.compucorp.civicrm.hrleaveandabsences/sql/auto_install.sql

```diff
@@ -134,7 +140,8 @@ INSERT INTO `civicrm_hrleaveandabsences_absence_type`(
   1,
   3,
   1,
-  1
+  1,
+  'Sickness'
```

Comment:
I was thinking there would be a new leave type category table that would then be a foreign key in the absence type table. This means that these categories, Leave, Sickness and TOIL will be system defined. Can they be modified by the admin via the UI? 
Also thinking of some properties we have on the absence type table like is_sick, is_toil that is used to identify what type if leave type it is. Wondering how we can merge this and probably do away with the ones we don't use so we don't keep adding columns to the table.